python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -91,10 +93,22 @@ def fetch_latest_resolvable_version_string(requirement:)
                 parse_updated_files
               end
             rescue SharedHelpers::HelperSubprocessFailed => e
+              retry_count ||= 0
+              retry_count += 1
+
+              if compilation_error?(e) && retry_count <= 2
```

Comment:
Did you spot cases that needed two retries or worth just doing one without build isolation?

Reply:
Yeah, maybe one is just fine. I cargo culted this from places where we retry net requests, but this should probably just work the first time or something else is up 👍